standard_theme/static/js/search.js

```diff
@@ -18,7 +18,7 @@ function renderResults() {
     var term = $('#rtd-search-form input[name="q"]').val();
     var base_url = location.href.substring(0, location.href.indexOf('/search/?') + 1)
     $.ajax({
-        url: "https://www.standard-search.default.opendataservices.uk0.bigv.io/v1/search?q=" + encodeURIComponent(term) + "&base_url=" + encodeURIComponent(base_url),
+        url: "https://www.live.standard-search.opencontracting.uk0.bigv.io/v1/search?q=" + encodeURIComponent(term) + "&base_url=" + encodeURIComponent(base_url),
```

Comment:
Why is the server name `www.live.…` instead of just `live.…`?

Reply:
The server name is live.st ..... but the website URL is www.live.sta .....

I didn't set this server up originally so I'm just guessing, but I'm guessing this follows our normal pattern for servers and how our salt libraries work. They are set up to make the default Ubuntu page served as the default site for the box, and then specific sites are served under specific URL's. 

This allows more than one site on a box, and while it's correct to say that doesn't apply to this specific box at the moment, we don't know what will happen in the future. 

Also it means that bots who just scan every web server by IP address only find the default site and don't find any real sites to annoy.

Reply:
I meant FQDN; anyhow, I added the CNAME `standard-search.open-contracting.org` to follow our practice of avoiding provider-specific FQDNs if possible, which we can use here instead of `www.` once appropriate configuration changes are pade.

Reply:
A CNAME is a good idea; as the work to move this server has already been done and to avoid scope creep in tickets can we make a new issue/Trello for that?

Reply:
Do we really need a new card, instead of changing 3 lines now? https://github.com/OpenDataServices/opendataservices-deploy/commit/0f6a7d499496d34f36dcb86b1ed8b0d24a20a4f8

For all other transfers, please make adding a CNAME (if appropriate) part of the transfer.

Reply:
Made card: https://trello.com/c/q5gRnAIz/65-use-standard-searchopen-contractingorg-instead-of-bigvio-fqdn